test/functional/lua/vim_spec.lua

```diff
@@ -719,6 +719,10 @@ describe('lua stdlib', function()
       { ' b  ', 'b' },
       { '\tc', 'c' },
       { 'r\n', 'r' },
+      { '', '' },
+      { ' \t \n', '' },
+      -- double whitespace <C-k>IS
+      { '　f　', 'f' },
     }
 
     for _, q in ipairs(trims) do
```

Comment:
Forgive me if I'm wrong but isn't this test only checking if `q[2]` is false or not, which means it never fails? Shouldn't it be `assert(q[2] == trim(q[1])`.

Reply:
@lucario387 you can overload easy in lua. So I haven't looked deeply into this tests "trusting" previous contributors. But seems you are right and it is just regular assert and it should be `eq` instead.